tests/suites/test_suite_pk.function

```diff
@@ -1569,3 +1700,220 @@ exit:
     PSA_DONE();
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PSA_CRYPTO_C */
+void pk_get_psa_attributes(int pk_type, int from_pair,
+                           int usage_arg,
+                           int to_pair, int expected_alg)
+{
+    mbedtls_pk_context pk;
+    mbedtls_pk_init(&pk);
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    psa_key_usage_t usage = usage_arg;
+
+    MD_OR_USE_PSA_INIT();
```

Comment:
Quite sure I'm missing something here, but why `MD_OR_USE_PSA_INIT` instead of just `PSA_INIT()`?

Reply:
Indeed. `mbedtls_pk_get_psa_attributes` is only provided if `MBEDTLS_PSA_CRYPTO_C` is enabled (or potentially `_CLIENT` but not as far as tests is concerned). Having the wrong guards worked because the function doesn't actually require the PSA API except for attribute setter/getters. But there's no point in testing the function without having called `psa_crypto_init`, even if it can work. It's only useful when PSA is in use.